package.json

```
@@ -7,8 +7,9 @@
     "changelog": "changelog eslint-plugin-security all > CHANGELOG.md",
     "test": "mocha test/**/*",
     "lint": "eslint .",
+    "format": "prettier --write .",
     "lint:fix": "eslint --fix .",
-    "cont-int": "npm test && npm run lint"
+    "cont-int": "npm test && npm run lint && format"
```

It doesn’t make sense to format as part of CI
```suggestion
    "cont-int": "npm test && npm run lint"
```